jest.config.json

```diff
@@ -1,7 +1,6 @@
 {
 	"preset": "jest-preset-angular",
 	"setupFilesAfterEnv": ["<rootDir>/setup-jest.ts"],
-	"globalSetup": "jest-preset-angular/global-setup",
```

Comment:
not needed anymore ? 🎉

Reply:
nope, after NG16, there is no more `ngcc` compilation step